nginx.tpl.conf

```
@@ -33,17 +33,37 @@ http {
     gzip_proxied any;
     gzip_types *;
 
+    map $cookie_lang $lang {
+        default 'en';
+        zh      zh;
+        ko      ko;
+    }
+
     server {
         listen 80 default_server;
```

```suggestion
        listen 80 default_server;
        
        port_in_redirect off;
```
See https://github.com/jibrelnetwork/coinmena-web/pull/131